playbooks/common/openshift-checks/health.yml

```diff
@@ -1,5 +1,7 @@
 ---
-- include: ../openshift-cluster/evaluate_groups.yml
+- include: ../openshift-cluster/std_include.yml
+  tags:
+  - always
```

Comment:
Cargo-culting. But I did learn that tags are actually being used on playbook runs, just not in the way I expected. They're used to exclude certain things (as opposed to `--tags` which is a whitelist). So I left well enough alone. But thinking about it, I don't see why that use case would need to have this tagged `always` either.